.github/workflows/FormatCheck.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         # TODO: Change the call below to
         #       format(".")
         run: |
-          julia  -e 'using Pkg; Pkg.add(PackageSpec(name = "JuliaFormatter", version="1.0.45"))'
+          julia  -e 'using Pkg; Pkg.add(PackageSpec(name = "JuliaFormatter", version="1.0.60"))'
```

Comment:
The docs need to be updated as well: https://trixi-framework.github.io/Trixi.jl/stable/styleguide/#automated-source-code-formatting